ministryofjustice/correspondence_tool_staff
# CT-922 minor changes from QA tests

Fixes the issues raised in QA testing from CT-922

*QA TEST*
Tested on dev.
- Case detail page - should be 'Date received at MOJ'
- My open cases - heading should be 'My open cases'
- Case creation - Remove question mark on 'Clearance required by DACU - Disclosure'
